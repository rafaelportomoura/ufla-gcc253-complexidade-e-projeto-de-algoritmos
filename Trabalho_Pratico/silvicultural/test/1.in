7  
0 6  
5 4  
7 5  
8 2  
6 1  
1 0  
2 3  
3  
1 1  
2 3  
3 1  
4  
1 1  
2 3  
3 1  
4 2 
0