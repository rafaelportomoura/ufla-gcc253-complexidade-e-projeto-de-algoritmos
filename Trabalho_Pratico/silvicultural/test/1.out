25.6  
6.5  
7.9  